### ISSUE TITLE
some problem with dataset and loss function~

### ISSUE THREAD

**Issue by Mintpcc** [wangruixing/DeepUPE#59]
hello~I am very interested in your article. Recently I am trying to retrain your model, but I have encountered some problems.
First， where can I get the training dataset used in this paper? 
Second，I don't know if my loss function and training method are consistent with you because I didn't get an ideal result.

**Comment by wangruixing** [comment]
You can contact me by email: rxwang@cse.cuhk.edu.hk, I will send you the code to help you reproduce the result.
